Source/WTF/wtf/text/StringImpl.h

```diff
@@ -323,6 +323,11 @@ class StringImpl : private StringImplShape {
     static WTF_EXPORT_PRIVATE Expected<CString, UTF8ConversionError> utf8ForCharacters(const LChar* characters, unsigned length);
     static WTF_EXPORT_PRIVATE Expected<CString, UTF8ConversionError> utf8ForCharacters(const UChar* characters, unsigned length, ConversionMode = LenientConversion);
 
+    template<typename Func>
+    static Expected<std::invoke_result_t<Func, Span<const char>>, UTF8ConversionError> tryGetUtf8ForCharacters(const Func&, const LChar* characters, unsigned length);
```

Comment:
This should be `tryGetUTF8…` I don’t know why other identifiers are capitalizing it `Utf`. It is easier to spot when it has the traditional all capital style.

Reply:
Updated.